General/Baselines.md

```diff
@@ -771,20 +771,21 @@ The repository parameter is a string that can takes different form in case we ha
 
 The repository parameter to load a project from github/gitlab/bitbucket takes this form:
 
-`{prefix}://{owner}/{projectName}:{version}/{subFolder}`
+`{prefix}://{option}:{owner}/{projectName}:{version}/{subFolder}`
 
 This snippet should be configured with:
 
 - `{prefix}`: This is host-specific:
   - `github` for github
   - `bitbucket` for bitbucket
   - `gitlab` for gitlab
+- `{option}`: Optional server host, for private git servers
```

Comment:
```suggestion
- `{optionalHostname}`: Optional server host, for private git servers
```

Reply:
Since it is a server host url, why not "optionalHostURL" ?